### DISCUSSION TITLE
Update Nuclei Engine 2.6.2

### DISCUSSION THREAD

**Discussion by qusaialhaddad** [projectdiscovery/nuclei#1841] [Q&A] [answered]
Hi, when I run the command nuclei -update , I am getting this error 

[FTL] Could not create runner: could not fetch latest release: context deadline exceeded

tried with go install @latest  command , still the version shows 2.6.2.

**Comment by ehsandeep** [comment, marked as answer]
@qusaialhaddad looks like an issue with go setup, you can try removing existing nuclei binary and try installing again with `go install`

Know the current location of nuclei with `which nuclei` and remove the file, and install again using `go install`

**Comment by ndeepak** [comment]
Tried go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest
.. Remove it from bin and system before first.
